apps/web/src/pages/Pools/IndexPoolPage/components/PoolTable.tsx

```diff
@@ -67,6 +67,21 @@ function dateAccessor(value: Maybe<string>, intl: IntlShape) {
   ) : null;
 }
 
+const fullNameCell = (pool: Pool, intl: IntlShape) => {
+  return (
+    <span>
+      {getFullNameHtml(pool.owner?.firstName, pool.owner?.lastName, intl)}
+    </span>
+  );
+};
+
+const classificationsCell = (pills: JSX.Element[] | null) => {
+  if (pills) {
+    return <span>{pills}</span>;
+  }
+  return <span />;
```

Comment:
I think returning null would be a better option that returning an empty `<span />`

Reply:
done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/5631/commits/42fe014afa9588f87a12b3880dce81c6fd9c00f2